operator_ui/src/connectors/redux/reducers/dashboardIndex.ts

```diff
@@ -0,0 +1,24 @@
+export interface State {
+  recentJobRuns?: string
+  jobRunsCount?: number
+}
+
+const initialState = {
+  recentJobRuns: undefined,
+  jobRunsCount: undefined
+}
+
+export type Action = { type: 'UPSERT_RECENT_JOB_RUNS'; data: any }
```

Comment:
Can we type `data` in reducers or is it too much effort for now and for this PR?